RELEASING.md

```diff
@@ -141,6 +141,16 @@ The following examples all use `-m1` as an example - this gets replaced with a l
 2. Eclipse OpenJ9 tag the commit in the "release" branch that they want to be the milestone level as e.g. `openj9-0.17.0-m1`
 3. OpenJDK extensions branches the `openj9` branch to create the release branch, called `openj9-0.nn.0`
 4. Ask someone in the extensions team to make the following modifications:
+   * If this is M2 then check for new jdk tag to merge into the release branch:
```

Comment:
Just because `M2` may not be familiar to the casual reader
```suggestion
   * If this is milestone 2 (m2) then check for new jdk tag to merge into the release branch:
```